### PR TITLE
feat: admin page for workers and db driven platform settings

### PR DESCRIPTION
**Pull request by plyr4**
see: https://github.com/go-vela/server/pull/1110

## admin/workers

Screenshot 2024-04-26 at 4 50 53 PM

## admin/settings

Screenshot 2024-04-26 at 4 50 46 PM

### CODE REVIEW

**Comment by plyr4** [comment]
Screenshot 2024-05-03 at 4 10 21 PM

- added "Before:" element to capture and show the previous value when something is updated.
- added "last updated by" to the bottom

File: src/elm/Pages/Admin/Workers.elm [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+                                Http.BadStatus statusCode ->
+                                    case statusCode of
+                                        401 ->
+                                            "No workers found"
+
+                                        _ ->
+                                            "No workers found, there was an error with the server (" ++ String.fromInt statusCode ++ ")"
+
+                                _ ->
+                                    "No workers found"
```
Comment (wass3rw3rk) [inline src/elm/Pages/Admin/Workers.elm:243]: curious about the `401` message. did you intentionally not call out auth?

**Review by wass3rw3rk — COMMENTED** [review]
couple small things.. looking good

File: src/elm/Pages/Admin/Workers.elm (hunk 1 above)
Comment (plyr4) [reply src/elm/Pages/Admin/Workers.elm:243]: not sure, i think @ecrupper wrote that bit. i'll plop a message in there, good catch TY

File: src/scss/_table.scss [hunk 2]
```diff
@@ -142,6 +142,18 @@ td .key .list-item {
...
+.table-base tr.-busy {
+  border-bottom: none;
+  border-left: 2px solid var(--color-yellow);
+}
+
+.table-base tr.-available,
+.table-base tr.-idle {
+  border-bottom: none;
+  border-left: 2px solid var(--color-green);
+}
```
[the remark this answers is not in the document]
Comment (plyr4) [reply src/scss/_table.scss:155]: sweet, yeah for sure, thanks for pointing that out. lemme tweak

File: src/elm/Pages/Admin/Workers.elm (hunk 1 above)
Comment (plyr4) [reply src/elm/Pages/Admin/Workers.elm:243]: lol i didnt mean to call him out, i was just pointing it out in case it was intentional

**Review by ecrupper — APPROVED** [review]

**Review by wass3r — APPROVED** [review]

**Review by wass3rw3rk — APPROVED** [review]

**Pull request merged**